code/src/components/RadioButton.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+
+import "./form.css";
+import "./radiobutton.css";
+
+const answersArray = ["YES", "NO"];
+
+const RadioButton = ({ radioText, radioAnswer, onAnswerChange }) => {
+
+  return (
+    <section className="section-container">
+      <h2>{radioText}</h2>
+      <div className="radio-btn-container">
+        {answersArray.map(item => {
+          return (
+            <label tabIndex="0" key={item} htmlFor={item}>
```

Comment:
Same here with the htmlFor, I think you can skip it sense the input field is inside the label tag?